tasks/tests/unit/test_backfill_commit_data_to_storage_task.py

```diff
@@ -68,7 +68,7 @@ def test_all_report_rows(self, mock_handle_single_row, dbsession):
         def mock_handle_single_row_return_side_effect(db_session, commit, report_row):
             if report_row.code is None:
                 return {"success": True, "errors": []}
-            if report_row.code is "local":
```

Comment:
converted "is" to == which is the proper way to do this comparison

It's in a script though so idk if it really changes anything

Reply:
This was already fixed after rebasing 🤷